demo/scripts/controls/ribbonButtons/contentModel/tableBorderApplyButton.ts

```diff
@@ -1,16 +1,20 @@
 import MainPaneBase from '../../MainPaneBase';
-import { applyTableBorderFormat, isContentModelEditor } from 'roosterjs-content-model-editor';
 import { RibbonButton } from 'roosterjs-react';
+import {
+    applyTableBorderFormat,
+    BorderOperations,
+    isContentModelEditor,
+} from 'roosterjs-content-model-editor';
 
-const TABLE_OPERATIONS = {
-    menuNameTableAllBorder: 'AllBorders',
-    menuNameTableNoBorder: 'NoBorders',
-    menuNameTableLeftBorder: 'LeftBorders',
-    menuNameTableRightBorder: 'RightBorders',
-    menuNameTableTopBorder: 'TopBorders',
-    menuNameTableBottomBorder: 'BottomBorders',
-    menuNameTableInsideBorder: 'InsideBorders',
-    menuNameTableOutsideBorder: 'OutsideBorders',
+const TABLE_OPERATIONS: Record<string, BorderOperations> = {
```

Comment:
@JiuqingSong Following up on [this comment](https://github.com/microsoft/roosterjs/pull/2146#discussion_r1364228960), Changing from enum to string literals makes is so that if the values are modified it also needs to be changed everywhere it is used.

I made TABLE_OPERATIONS a Record to prevent the issue of all operations not working, but it is still the case that changing the value of a string literal makes it harder to modify everywhere than an enum.

Just an observation.